client/src/components/TournamentRow.tsx

```diff
@@ -10,11 +11,23 @@ export enum TournamentStatus {
   Finished,
 }
 
-export function TournamentRow(props: { tournament: Tournament }): JSX.Element {
+export function TournamentRow(props: { tournament: TournamentRecord }): JSX.Element {
+  const startDate: Date = new Date(props.tournament.start_date)
+
   return (
     <Container>
       <Card>
-        Name: {props.tournament.name} (Status: {TournamentStatus[props.tournament.status]})
+        <Typography>
+          Name: {props.tournament.name} (Status: {props.tournament.status_id})
+        </Typography>
+        {props.tournament.description ? (
+          <Typography>{props.tournament.description}</Typography>
+        ) : (
+          ''
+        )}
+        <Typography>
+          Start Date: {startDate.toUTCString()} (in <CountdownTimer deadline={startDate} />)
```

Comment:
Maybe use `startDate.toLocaleString()` instead? I think it's easier to understand. What do you think?

I'm running en-US at UTC+1, so for the same date:

LocaleString | UTCString
------------ | -------------
2/25/2020, 9:30:00 AM | Tue, 25 Feb 2020 08:30:00 GMT

Reply:
Yeah, I had that at first and changed it to UTCString for some reason. Never changed it back :D